### ISSUE TITLE
Can't seem to grow Industrial Hemp under artificial light

### ISSUE THREAD

**Issue by Techokami** [BluSunrize/ImmersiveEngineering#250]
So I have an underground farming complex, growing various plants both vanilla and mod-added, using vanilla farming methods. Everything grows correctly... except for Industrial Hemp. Which I don't have an alternative for. (Yes, AgriCraft is present, but I've not bothered with that tech tree since vanilla farming isn't disabled and I'm not shooting for Botania stuff, which is gated by AgriCraft in the pack I'm using)

Does Industrial Hemp require an unobstructed view of the sky to grow? And if so, why?

**Comment by BluSunrize** [comment]
It needs a lightlevel of 12 or higher, and gets a big boost in growth speed when it can see the sky.

**Comment by Techokami** [comment]
So, if it's underground, and has a ceiling made entirely of glowstone 4 blocks above the farmland, that's not going to cut it? :confused:

**Comment by BluSunrize** [comment]
I honestly don't know, never tried it .~.
Keep experimenting, but ideally you should have your farm above ground

**Comment by Techokami** [comment]
Ideally, yes. Except the only place I was able to set up shop on my server was between a mountain *and another mountain*. That, and there was a humongous cavern underneath, so I used what I had available.

That, and the majority of crops in Minecraft, both vanilla and mod-added, seem to use a light level of 9 as the minimum for growth. For Industrial Hemp, I was able to get SOME progress by placing Glowstone directly next to the plants... but now it looks funky to have two rows of Glowstone instead of irrigation canals. :unamused: 

I might PR an adjustment.
